docs/src/grids.md

```diff
@@ -583,3 +582,311 @@ grid = RectilinearGrid(architecture, float_type,
 
 For more examples see [`RectilinearGrid`](@ref Oceananigans.Grids.RectilinearGrid)
 and [`LatitudeLongitudeGrid`](@ref Oceananigans.Grids.LatitudeLongitudeGrid).
+
+## Distributed grids
+
+!!! note
+    For the following examples, make sure to have both
+    `Oceananigans` and `MPI` in your
+    [environment](https://pkgdocs.julialang.org/v1/environments/).
+
+Next we turn to the distribution of grids across disparate nodes.
+This is useful for running simulations that cannot fit on one node.
+It can also be used to speed up a simulation -- provided that the simulation 
+is large enough such that the added cost of communicating information between
+nodes does not exceed the benefit of dividing up the computation among different nodes.
+
+```julia
+# Make a simple program that can be written to file
+make_distributed_arch = """
+
+using Oceananigans
+using MPI
+MPI.Init()
+    
+architecture = Distributed()
+on_rank_0 = MPI.Comm_rank(MPI.COMM_WORLD) == 0
```

Comment:
```suggestion   
architecture = Distributed()
on_rank_0 = architecture.local_rank == 0
```
we can also remove all the MPI stuff inside the script. It should be initialized by default. I am ok with keeping it though because it teaches a bit about MPI

Reply:
Oh I just noticed I commented the same thing below 😅 .

Reply:
What's the value of teaching about MPI though? This code is for fluid dynamics...